src/Cemu/Tools/DownloadManager/DownloadManager.cpp

```diff
@@ -371,7 +371,7 @@ bool DownloadManager::syncAccountTickets()
 	for (auto& tiv : resultTicketIds.tivs)
 	{
 		index++;
-		std::string msg = _("Downloading account ticket").utf8_string();
+		std::string msg = _("Downloading account ticket").ToStdString();
```

Comment:
In this file I'm just reverting what I did in #956, since I couldn't verify whether those specific strings were displayed correctly or not.